deepchecks/base/check_context.py

```diff
@@ -43,8 +43,8 @@ class CheckRunContext:
         timeout in second for the permutation features importance calculation
     scorers : Mapping[str, Union[str, Callable]] , default: None
         dict of scorers names to scorer sklearn_name/function
-    non_avg_scorers : Mapping[str, Union[str, Callable]] , default: None
-        dict of scorers for multiclass without averaging of the classes
+    scorers_per_class : Mapping[str, Union[str, Callable]] , default: None
+        dict of scorers for classification without averaging of the classes
```

Comment:
WDYT about adding a link to sklearn - to exemplify these types of scorers...?
(or any other way that gives an example of "what is this scorer", because I don't think it's intuitive for the user...
and maybe it shouldn't be documented here cause this isn't really user facing, but was thinking "where to start")

Reply:
The same parameters here will be exposed to the user (in suite or check run) so we'll just copy the docstring anyway